src/main.rs

```diff
@@ -246,14 +274,30 @@ struct ReplayArgs {
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let opt = Cli::parse();
-    let filter = EnvFilter::from_default_env();
+
+    let log_level_filter = LevelFilter::from(opt.log);
+    CombinedLogger::init(vec![
+        TermLogger::new(
+            log_level_filter,
+            Config::default(),
+            TerminalMode::Mixed,
+            ColorChoice::Auto,
+        ),
+        WriteLogger::new(
+            log_level_filter,
+            Config::default(),
+            File::create("era_test_node.log").unwrap(),
```

Comment:
so it will create this file in the place where you run the binary?

Reply:
Currently, yes.

Reply:
Can we make this configurable and avoid hardcoding the file name? Being able to specify a different log file path when we run the node with the `--log_file_path` argument or something similar.

Reply:
And please update the `.gitignore` with wildcard for .log please.